_integration-schemas/codat/v1/bank_statements.md

```diff
@@ -12,8 +12,8 @@ description: |
 replication-method: "Full Table"
```

Comment:
This rep method isn't correct. [Take a look here](https://github.com/singer-io/tap-codat/blob/master/tap_codat/streams.py#L412).